webextension/background.js

```diff
@@ -129,8 +127,75 @@ const Config = (function() {
       }
     }
 
-    load() {
+    maybeSendTelemetry(message) {
+      browser.study.sendTelemetry(message);
+    }
+
+    get shieldStudySetup() {
+      const pref = `extensions.${browser.runtime.id.split("@")[0]}.variation`;
+      return {
+        activeExperimentName: browser.runtime.id,
+        allowEnroll: true,
+        studyType: "shield",
+        telemetry: {
+          send: true,
+          removeTestingFlag: !this._testingMode,
+        },
+        endings: {
+          "user-disable": {
+            baseUrl: null,
+          },
+          expired: {
+            baseUrl: null,
+          },
+          dataPermissionsRevoked: {
```

Comment:
Revoked is delayed until 5.1, because normandy is *supposed* to do this.  They will come in as 'user-disable'

Reply:
Ah, then I'll remove that one. Thanks!